public/index.html

```diff
@@ -45,13 +52,13 @@
                             <p class="heading">TÄISNIMI</p>
                             <p class="content">{{currentClimber.name}}</p>
                         </div>
-                        <div class="row">
+                       <div class="row">
                             <p class="heading">EKSAMI AEG</p>
-                            <p class="content">2020/23/02</p>
+                            <p class="content">{{currentClimber.examTime?.replaceAll('-','/')}}</p>
```

Comment:
A OK quick solution imho for now, but the client should actually parse this and then format a proper date as a string.

I really like the direction you're going with putting the calculated values into the component side - should we do that here as well for now?

Reply:
Moved the property onto the currentClimber object, seemed better than scattering the climber info, let me know if that makes sense. Yeah, took the lazy route right now - could not remember of the top what the correct way to force region (so that it does not follow the browser) was with the js datetime